src/Compute/Compute/Generated/GalleryImage/GalleryImageCreateOrUpdateMethod.cs

```diff
@@ -185,7 +188,19 @@ public override void ExecuteCmdlet()
 
                     if (this.IsParameterBound(c => c.Feature))
                     {
-                        galleryImage.Features = this.Feature;
+                        galleryImage.Features = new List<GalleryImageFeature>();
+                        for (int i = 0; i < this.Feature.Length; i++)
+                        {
+                            galleryImage.Features.Add(this.Feature[i]);
+                        }
+                    }
+                    if (SecurityTypeNotFoundInFeatures(galleryImage.Features))// if SecurityType is not in Features, default to TrustedLaunch
```

Comment:
If the user sets HyperVGen to V1 , should this defaulting still happen? I don't think so.

Reply:
thanks for pointing this out. confirmed w feature team and HyperVGen V2 is indeed required for TL

Reply:
also verified with feature team,  HyperVGeneration will default to Gen2, regardless of SecurityPosture

Reply:
code is updated to reflect, no security type defaulting if Gen1 HyperV is set